client/core/account.go

```diff
@@ -250,11 +245,10 @@ func (c *Core) UpdateCert(host string, cert []byte) error {
 
 	acct.Cert = cert
 	dc, err := c.connectDEX(acct)
-	if dc == nil {
-		return fmt.Errorf("failed to connect using new cert (will attempt to restore old connection): %v", err)
-	}
 	if err != nil {
-		dc.connMaster.Disconnect() // stop any retry loop for this new connection.
+		if dc == nil {
```

Comment:
Should be `if dc != nil`.

Reply:
Darn, I missed this. Thanks for the review.